libnmstate/nm/connection.py

```diff
@@ -137,36 +137,83 @@ def commit_profile(connection_profile, save_to_disk=True, nmdev=None):
     con_profile.commit(save_to_disk, nmdev)
 
 
+class ConnectionSetting(object):
+
+    def __init__(self, con_setting=None):
+        self._setting = con_setting
+
+    def create(self, con_name, iface_name, iface_type):
+        con_setting = nmclient.NM.SettingConnection.new()
+        con_setting.props.id = con_name
+        con_setting.props.interface_name = iface_name
+        con_setting.props.uuid = str(uuid.uuid4())
+        con_setting.props.type = iface_type
+        con_setting.props.autoconnect = True
+        con_setting.props.autoconnect_slaves = (
+            nmclient.NM.SettingConnectionAutoconnectSlaves.YES)
+
+        self._setting = con_setting
+        self._logging_connection_info('create_setting')
```

Comment:
the logging message needs to be adjusted

Reply:
You mean changing it to `create`?

Reply:
maybe `ConnectionSetting.create() done`